requirements.txt

```diff
@@ -12,11 +12,12 @@ ghp-import==2.0.2
 idna==3.3
 importlib-metadata==4.8.1
 importlib-resources==5.4.0
+iniconfig==1.1.1
 isodate==0.6.0
 Jinja2==3.0.2
 joblib==1.1.0
+jsonpath-ng==1.5.3
```

Comment:
Are you sure the package name is `jsonpath-ng` and not `jsonpath_ng`?

Reply:
Yes: https://pypi.org/project/jsonpath-ng/
Actually, I do the update of the requirements file with `pip3 freeze`. This gets usually the correct names and versions.